quic/s2n-quic-transport/src/stream/tests/mod.rs

```diff
@@ -19,6 +19,7 @@ macro_rules! assert_matches {
         }
```

Comment:
Can we rename this file to `testing.rs` instead of `tests/mod.rs`? Usually testing utilities are in the `testing` module whereas actual tests are in `tests`.

There shouldn't be anything else in this directory, right?